www/assets/js/components/SearchPage.tsx

```diff
@@ -157,6 +157,37 @@ export default function SearchPage(props: SearchPageProps) {
     history
   )
 
+  const scrollChildRef = useRef<HTMLDivElement>(null)
+  /**
+   * InfiniteScroll has a bug where:
+   *  - if initial data does not fill the scroll area, or
+   *  - scroll area is resized so that it is no longer filled by scroll data
+   * then no new request is made.
+   *
+   * This is a work-around to address that.
+   *
+   * This returns a ref to be attached to the child of InfiniteScroll that fixes
+   * the issue.
+   *
+   * See https://github.com/danbovey/react-infinite-scroller/issues/294
+   */
+  const checkIfNeedMore = useCallback(() => {
```

Comment:
We have an issue on OCW before this PR where
- IF search page fits 10+ results on initial load (or after resize)
- THEN new results are never loaded.

That's not very realistic (you have to zoom waaaaay out to make it happen on ocw currently).

But this PR exacerbates the issue because no longer do 10 items need to fit on the visible search area at once, 10 items just need to fit within 150% of the viewport height at once (see CSS changes above). 

So I decided to address it.

We could also increase the search page size a bit, which would make the workaround here less relevant.